File: get_columns.py
import urllib.request
import json

url = 'http://93.127.212.235:32770/api/v2/meta/tables/mwby85581fhjy27'
headers = {'xc-token': 'dZMS2te8v6cf47Jlmlnk3S3ft9LT_QO8bjNdOcZZ'}

req = urllib.request.Request(url, headers=headers)
try:
    with urllib.request.urlopen(req) as response:
        data = json.loads(response.read().decode())
        for column in data['columns']:
            print(f"{column['title']}: {column['column_name']}")
except Exception as e:
    print(f"Error: {e}")

File: get_columns.js
const fs = require('fs');
const rawData = fs.readFileSync('client_meta_raw.json', 'utf8');
const data = JSON.parse(rawData);
let out = '--- DETAILED COLUMNS ---\n';
data.columns.forEach(c => {
    out += `Title: "${c.title}" | Column: "${c.column_name}" | Type: ${c.dt} | PK: ${c.pk} | Req: ${c.rqd || false}\n`;
    if (c.colOptions) {
        out += `  Options: ${JSON.stringify(c.colOptions.options || c.colOptions)}\n`;
    }
});
fs.writeFileSync('column_details.txt', out);
console.log('Done writing column_details.txt');
